### ISSUE TITLE
Link to and look at

### ISSUE THREAD

**Issue by mhulse** [mhulse/dotfizzles#52]
https://github.com/dotfiles/dotfiles.github.com

Anything there worth copying?

**Comment by beekman** [comment]
Yep! https://github.com/atomantic/dotfiles. It uses ZSH but I think it's worth looking at what he's doing that you might want to copy.
